dashboard/app/views/layouts/_header.html.haml

```diff
@@ -12,6 +12,15 @@
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
 .header-wrapper.hide_on_mobile{class: ('feedback-bug-links' if show_bug_links)}
+  - if current_user && script_level
```

Comment:
here you can do
````
  - if current_user && script_level && script
````
then you don't need the tristate below, and then I would eliminate the local `post_milestone` var